### DISCUSSION TITLE
Visual mode select un-sequential

### DISCUSSION THREAD

**Discussion by Felipe-9** [sxyazi/yazi#1581] [1. Q&A] [answered]
Hello there, this might sound silly but i havent figured out to select files using visual mode like the video in yazi features page:

https://yazi-rs.github.io/videos/visual-mode_bulk-rename.mp4

<video width="320" height="240" controls>
  <source src="https://yazi-rs.github.io/videos/visual-mode_bulk-rename.mp4" type="video/mp4">
</video>

how is it done?

**Comment by xfzv** [comment, marked as answer]
1. Press `v` to enter visual mode
2. Select multiple files with either `j` or `down`
3. Exit visual mode with `<ESC>`
4. Files are now marked with yellow color instead of green
5. Move down with either `j` or `down`
6. Repeat 1-3 again and so on

**Reply by Felipe-9** [reply]
thanks, i knew it would be something easy that i just couldnt figure it out.
